config/default.yml

```diff
@@ -1,25 +1,36 @@
 # This is the default configuration file.
 
 Packaging/BundlerSetupInTests:
-  Description: 'Avoid using `bundler/setup` in your tests.'
+  Description: >-
+                 Using `bundler/setup` in tests is redundant. Consider
+                 removing it.
   Enabled: true
   VersionAdded: '0.4'
   VersionChanged: '0.5'
 
 Packaging/GemspecGit:
-  Description: 'Use pure Ruby alternative instead of `git ls-files`.'
+  Description: >-
+                 Using `bundler/setup` in tests is redundant. Consider
```

Comment:
You probably didn't mean to mention `bundler/setup` here.

Reply:
Aw, crap! I forgot this here 🤦🏻 
Would you have time to just make a straight-forward commit fixing this?

Reply:
Just btw, this is a pretty impressive catch, thanks! 😄

Reply:
Sorry, I'm a bit buried right now, I'm just quickly going through notifications and saw this, so just wanted to let you now.

Reply:
No problem, fixed via a9925e7f7e6624e961965876684f5c0ebb05f487.